serverless.yml

```diff
@@ -708,6 +722,12 @@ resources:
 
   Resources:
 
+    AutoRemediateQUEUE:
+      Type: AWS::SQS::Queue
+      Properties: 
+        MessageRetentionPeriod: 7200 #2 Hours
+        QueueName: CloudConformityAutoRemediate
+
```

Comment:
The SQS queue must be encrypted at rest. Please configure KMS KeyId in the queue property